utils/json_requests

```diff
@@ -0,0 +1,82 @@
+import sys
+import logging
+import json
+import requests
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+"""
+Running command:
+python json_requests.py **path to dialogues_ems.json/empty string **  **intents to post separated by spaces/empty string***
+"""
+
+def post_intent(intents: str, url:str , auth_token: str)-> None:
+    """
+    :param intents: A list of all intents to be added to the app
+    :return:
+    """
+    logger.debug(f"******Add intents to wit app********")
+
+    header = {'Authorization': 'Bearer ' + auth_token, 'content-type': 'application/json'}
+
+    for i,intent in enumerate(intents):
+        data = {"name": intent}
+        response = requests.post(url, json=data, headers=header)
+        logger.debug(f"Response for intent {i} : {response.json()}")
+
+
+
+def post_utterrances(file , url , auth_token ):
+    """
+    :param json_file: File name from folder data to post utterances for training
+    :return:
+    """
+    logger.debug(f" ******** Training for the utterrances *********")
+
+    payload  = []
+    header = {'Authorization': 'Bearer ' + auth_token, 'content-type': 'application/json'}
+
+    with open(file) as json_file:
+        data = json.load(json_file)
+        for d in data:
+            temp_dict = {}
+            temp_dict["text"] = d["utterance"]
+            temp["intent"] = d["intent"]
+            temp["entities"] = []
+            temp["traits"] = []
+            payload.append(temp_dict)
+
+    response = requests.post(url, json=data, headers=header)
+    logger.debug(f"Response for training:\n {response.json()}")
+
+    logger.debug(f"*********Training complete**********")
```

Comment:
Space the asterisk and the words, like line 35

Reply:
Fixed in the new commit !